web/modules/custom/dept_migrate/modules/dept_migrate_group_files/config/install/migrate_plus.migration.d7_file.yml

```diff
@@ -10,13 +10,12 @@ migration_tags:
 migration_group: migrate_group_files
 label: 'Public files'
 source:
-  plugin: d7_file
+  plugin: d7_file_uuid
   scheme: public
   batch_size: 1000
   constants:
     source_base_path: 'https://www.northernireland.gov.uk'
 process:
-  fid: fid
```

Comment:
Removed as we want to rely on autoincrement + use UUID instead for tracking content.